stream-chat-android-ui-components/src/main/res/values-ko/strings_attachment_gallery.xml

```diff
@@ -15,8 +15,8 @@
     limitations under the License.
 -->
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-  <string name="stream_ui_attachment_gallery_count">%1$d / %2$d</string>
-  <string name="stream_ui_attachment_gallery_date">"%1$s %2$s에 보냈습니다"</string>
+  <string name="stream_ui_attachment_gallery_count">"%1$d / %2$d"</string>
+  <string name="stream_ui_attachment_gallery_date">"\"%1$s %2$s에 보냈습니다\""</string>
```

Comment:
@adasiewiczr I updated LocaLazy yesterday but it seems not to be applied 🤔

Reply:
It seems that GH-Action ran before you completed translations.
Lets wait until tonight that the GH-Action will run again, providing new updates